.travis.yml

```diff
@@ -10,23 +10,55 @@ script:
   - script/build
 xcode_workspace: ReactiveCocoa.xcworkspace
 matrix:
+  allow_failures:
+    - xcode_scheme: ReactiveCocoa-iOS
+      osx_image: xcode8
```

Comment:
Because the iOS test on Xcode 8 is now randomly failing as seen in https://github.com/Quick/Nimble/pull/302#issuecomment-226390685 and https://github.com/Quick/Quick/pull/547.